bitnami/zookeeper/README.md

```diff
@@ -236,11 +236,12 @@ services:
   ...
 ```
 
-### Configuration
+### Zookeeper Configuration
```

Comment:
The capitalization should be `ZooKeeper`
```suggestion
### ZooKeeper configuration
```

It is wrong in other places in the file though and maybe it is not important to fix it in this PR.